nncf/quantization/algorithms/post_training/algorithm.py

```diff
@@ -136,28 +138,19 @@ def _create_statistics_aggregator(self,
             return ONNXStatisticsAggregator(dataset)
         return None
 
-    def _get_prepared_model_for_compression(self, model: TModel, backend: BackendType) -> TModel:
-        if backend == BackendType.ONNX:
-            from nncf.experimental.onnx.model_normalizer import \
-                ONNXModelNormalizer
-            return ONNXModelNormalizer.normalize_model(model, convert_opset_version=True)
-
-        return None
-
     def _apply(self,
                model: TModel,
                statistic_points: Optional[StatisticPointsContainer] = None,
                dataset: Optional[Dataset] = None) -> TModel:
 
-        modified_model = model
+        modified_model = deepcopy(model)
```

Comment:
Could you briefly explain why you are doing a deep copy?

Reply:
I asked the same question.
Answer was:
> The user doesn't expect that the original model that he passed will be changed, I suppose.
We should work on a copy of that.